src/SPC/command/CraftCommand.php

```diff
@@ -104,6 +104,25 @@ public function handle(): int
         // craft build
         if ($craft['craft-options']['build']) {
             $args = [$extensions, "--with-libs={$libs}", ...array_map(fn ($x) => "--build-{$x}", $craft['sapi'])];
+            foreach ($craft['build-options'] as $option => $value) {
+                if ($value === false) {
+                    continue;
+                }
+                if ($value === true) {
+                    $args[] = "--{$option}";
+
+                    continue;
+                }
+                if (is_array($value)) {
+                    foreach ($value as $subVal) {
+                        $args[] = "--{$option}={$subVal}";
+                    }
+
+                    continue;
+                }
+
+                $args[] = "--{$option}={$value}";
+            }
```

Comment:
https://github.com/crazywhalecc/static-php-cli/blob/ca3d6d14d0b8df5ad04b881fbdaa50f6ad661a3f/src/SPC/command/CraftCommand.php#L83-L95

Seems we could extract this option parsing in a separate function, just re-use this section.